netsim/tools/nuts/inventory/hosts.j2

```diff
@@ -0,0 +1,49 @@
+{% set napalm_platform = {
```

Comment:
You could move this to `inventory.napalm` tool parameter (like all defaults, the tool parameters are not schema-checked). It would allow people to add new devices via system defaults and you could check whether a device is supported when generating tests.

Reply:
Thanks for this tip. Did not know it was so easy. This made a lot of stuff easier and nicer

Reply:
is now `defaults.tools.nuts.inventory.napalm`